api/src/main/java/io/strimzi/api/kafka/model/KafkaExporterResources.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+/**
+ * Encapsulates the naming scheme used for the resources which the Cluster Operator manages for a
+ * {@code KafkaExporter} instance.
+ */
+public class KafkaExporterResources {
+    protected KafkaExporterResources() { }
+
+    /**
+     * Returns the name of the Kafka Exporter {@code Deployment}.
+     * @param kafkaClusterName  The {@code metadata.name} of the {@code KafkaExporter} resource.
```

Comment:
it's the Kafka cluster name so `metadata.name` of the `Kafka` resource. It should be fixed for all the other methods.